CMakeLists.txt

```diff
@@ -119,15 +135,6 @@ function(curl_dumpvars)  # Dump all defined variables with their values
   message("::endgroup::")
 endfunction()
```

Comment:
I suggest moving the above function right below the empty line after the first `endforeach()` line in the file.
This allow using this function to debug the version extraction.